airflow/providers/sqlite/db/sqlite.py

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+import typing
+
+from airflow.providers.sqlite.hooks.sqlite import SqliteHook
+
+if typing.TYPE_CHECKING:
+    from airflow.providers.common.sql.hooks.sql import DbApiHook
+
+schemes = ["sqlite", "sqlite3"]
```

Comment:
We already have this info in the provider.yaml:

```yaml
connection-types:
  - hook-class-name: airflow.providers.sqlite.hooks.sqlite.SqliteHook
    connection-type: sqlite
```

It would be nice to only have this info in one place

Reply:
Yeah I just realised this as well. Will switch to use that instead.

This kind of raises another thing for @bolkedebruin—should filesystem protocol definitions be moved from `fs.py` to `provider.yml` instead?

Reply:
Probably, yes. Having the info be discoverable without needing to import code is preferable

Reply:
> Yeah I just realised this as well. Will switch to use that instead.
> 
> This kind of raises another thing for @bolkedebruin—should filesystem protocol definitions be moved from `fs.py` to `provider.yml` instead?

It might make sense, but I'd like to hold off on that one for a bit. I intend to extend the filesystem specs with Arrow which would make for fallback scenarios which I do not know the `provider.yml` would support.